spec/base_decorator_spec.rb

```diff
@@ -0,0 +1,16 @@
+require_relative '../classes/base_decorator'
+
+describe Decorator do
```

Comment:
- Kindly check your test running `rspec .` there are some errors in the bash. Including variable name errors like in this line

![image](https://github.com/Progress2002/OOP-school-library/assets/93630700/321f3521-98f7-42e3-bebf-db8ea452bec6)